server/src/file-storage/file-storage.module.ts

```diff
@@ -0,0 +1,20 @@
+import { Module } from '@nestjs/common';
+import { FilesStorageService } from './file-storage.service';
+import {
+  AzureBlobStorage,
+  AZURE_BLOB_PROVIDER_TOKEN,
+} from './storage-providers/azure-blob-storage';
+import {
+  LocalStorageProvider,
+  LOCAL_STORAGE_PROVIDER_TOKEN,
+} from './storage-providers/local-storage';
+
+@Module({
+  providers: [
+    { provide: AZURE_BLOB_PROVIDER_TOKEN, useClass: AzureBlobStorage },
+    { provide: LOCAL_STORAGE_PROVIDER_TOKEN, useClass: LocalStorageProvider },
```

Comment:
This module adds these 2 classes as providers, so the `FileStorageService` can import them. All other modules importing the `FileStorageModule` will get the `FileStorageService` with the providers.